srcpkgs/chicken/template

```diff
@@ -1,13 +1,13 @@
 # Template file for 'chicken'
 pkgname=chicken
-version=4.12.0
-revision=2
+version=5.0.0
+revision=1
 short_desc="A practical and portable Scheme system"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
 license="BSD"
```

Comment:
Please change this to an spdx identifier. BSD-2-Clause or BSD-3-Clause, most likely. Use `git commit --amend` to add changes to the most recent commit without creating a new commit, then force push again.

Otherwise looks good.